### DISCUSSION TITLE
Data import with special characters in the password

### DISCUSSION THREAD

**Discussion by MeinhardSchneider** [dani-garcia/vaultwarden#4405] [General]
Hello,

I have a problem with the data import.

So far I have used LastPass and would like to switch to Vaultwarden. With LastPass I have exported all entries (the only format seems to be CSV).

Now I have the problem that my passwords always come from the password generator and therefore also contain special characters such as ',', '"', ''', '>', '<', '\' etc..
Especially the comma is of course a big problem in a CSV file. If I try to import the export with LastPass (~500 entries), all entries with passwords that contain a comma are completely mixed up because the CSV file is split up incorrectly.
So I can't avoid modifying the CSV file. Is there a way to quote the comma so that it is not recognized as a field separator? Alternatively, I would have to write a script that converts the CSV file to JSON - but then I would have to quote the characters '<' and '>'.

Are there other solutions to the problem?

Many thanks & best regards
Meinhard

**Comment by BlackDex** [comment]
First, Vaultwarden it self doesn't handle the conversion of those imports, that is all done by the clients.
Second, you should be able to put quotes `"` around all values/columns which should prevent `,` or other special variables from meaning something else then being part of the value.

Also, read https://bitwarden.com/help/import-from-lastpass/
